tests/components/unifiprotect/conftest.py

```diff
@@ -217,6 +217,7 @@ def doorbell_fixture(camera: Camera, fixed_now: datetime):
         SmartDetectObjectType.PERSON,
         SmartDetectObjectType.VEHICLE,
         SmartDetectObjectType.ANIMAL,
+        SmartDetectObjectType.PACKAGE,
```

Comment:
Note for other reviewers: this is why is the switch and binary sensor platform counts in the other tests are up by one now:
- package binary sensor 
- package detection on/off switch